backend/config/services.yaml

```diff
@@ -72,7 +72,11 @@ services:
     App\Admin\ResetPasswordRequestAdmin:
         arguments: [null, App\Entity\ResetPasswordRequest, null]
         tags:
-            - { name: sonata.admin, manager_type: orm, label: Reset Password Requests }
+            - {
```

Comment:
i would revert this change, we prefer to add tags the older way (there are multiple cases in this file)

Reply:
Ok, anyway, to keep it controlled I have added a longer max line length in editorconfig